src/modules/filter-sort/helpers/filter-by-search.js

```diff
@@ -33,7 +33,7 @@ export default function (search, keys, items) {
       }))
 
     if (matchedSearch) {
-      return [...p, i]
+      return [...p, items[i].id]
```

Comment:
This file appears to have no test coverage. Please add some for the bits you changed.